docs/articles/vs-test-adapter/Adapter-Release-Notes.md

```diff
@@ -1,5 +1,39 @@
 # Adapter Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 3.17.0 - July 11, 2020
+
+This release has a series of enhancements and bugfixes.  There have been two 3.17 beta releases (beta.1 and beta.2) published earlier on nuget.org.
+A major bugfix is that the console output now is properly coming out.  There are also a series of fixes for corner cases.  
+
+[StopOnError](https://docs.nunit.org/articles/vs-test-adapter/Tips-And-Tricks.html#stoponerror) can be very useful to reduce time of long test runs that are failing.
+
+The ability to set custom paths for the test code can be useful with data driven tests, where you want to point to the data file (and line number therein) instead of usually trivial test case.
```

Comment:
What does "instead of usually trivial test case" part mean?

Reply:
Changed.....